eadk_discord/__main__.py

```diff
@@ -1,34 +1,33 @@
 # pragma: coverage exclude file
 import logging
 import os
+import sys
+import tomllib
 from pathlib import Path
 
-import discord
-from discord.abc import Snowflake
-
 from eadk_discord import bot_setup
 
-if __name__ == "__main__":
-    logging.basicConfig(level=logging.DEBUG)
 
-    bot_token_option: str | None = os.getenv("DISCORD_BOT_TOKEN")
-    if bot_token_option is None:
-        raise ValueError("DISCORD_BOT_TOKEN is not set in environment variables")
-    else:
-        bot_token: str = bot_token_option
+def get_config_path(env_var_name: str) -> str:
+    match len(sys.argv):
+        case 1:
+            env_var_option: str | None = os.getenv(env_var_name)
+            if env_var_option is None:
+                raise ValueError(f"environment variable '{env_var_name}' is not set")
+            else:
+                return env_var_option
+        case 2:
+            return sys.argv[1]
+        case num_args:
+            raise ValueError(f"expected 0 or 1 argument, got {num_args-1}")
```

Comment:
nobody manually inspects `sys.argv` nowadays, once you're used to it it's very quick to use https://docs.python.org/3/library/argparse.html